package.json

```diff
@@ -15,6 +15,7 @@
   "engines": {
     "vscode": "^1.79.0"
   },
+  "packageManager": "pnpm@10.0.0",
```

Comment:
Would it be best to lock just the major version, e.g. `pnpm@10`?

Reply:
I would prefer that option and tried it, but unfortunately they require a full `X.Y.Z` version to be specified.